test/jdk/sun/security/krb5/name/Constructors.java

```diff
@@ -25,7 +25,7 @@
  * @bug 6966259
  * @summary Make PrincipalName and Realm immutable
  * @modules java.security.jgss/sun.security.krb5
- * @run main/othervm Constructors
```

Comment:
Do you know why this test needs the change? It's not doing much (no blocking calls) and on my system runs in about a tenth of a second.

Reply:
I have not analysed why. I can see 1013 occurrences in our database where the tests takes 2 minutes or more. On different CPU platforms.